JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -4,9 +4,48 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
-
-// 과일 저장소 타입
 class FruitStore {
+    typealias FruitsInventory = [Fruit: Int]
+    private(set) var fruitsInventory: [Fruit: Int] = [:]
+    
+    init() {
+        Fruit.allCases.forEach {
+            fruitsInventory[$0] = 10
+        }
+    }
+    
+    func increaseInventory(about fruitsInventory: FruitsInventory) throws {
```

Comment:
```suggestion
    func increaseInventory(about inventory: FruitsInventory) throws {
```
inventory 라는 단어가 중복적으로 많이 쓰이는 것 처럼 보여요 ㅎㅎ 그래서 메서드가 조금 길어진것 같네요 
의미전달은 충분해서 고친다면 요정도로 줄여도 무관할 것 같아요 ㅎㅎ (지금도 충분합니다)